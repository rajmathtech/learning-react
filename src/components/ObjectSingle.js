import React from 'react';
import {Link} from 'react-router-dom';
const ObjectSingle = (props) => (
    <div className="singleObject"> 
        <Link to={`display/${props.object.id}`} className="objectLink" >
            {props.object.title.toUpperCase()}
        </Link>
    </div>
);
export default ObjectSingle;