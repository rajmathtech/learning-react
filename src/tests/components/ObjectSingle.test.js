import React from 'react';
import {shallow} from 'enzyme';
import objects from '../fixtures/objects';
import ObjectSingle from '../../components/ObjectSingle';
test('should render component', () => {
    const wrapper = shallow(<ObjectSingle object={objects[1]}/>);
    expect(wrapper).toMatchSnapshot();   
});