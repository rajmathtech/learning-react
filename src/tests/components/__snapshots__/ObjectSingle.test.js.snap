// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div
  className="singleObject"
>
  <Link
    className="objectLink"
    replace={false}
    to="display/2"
  >
    OBJECT TWO
  </Link>
</div>
`;
